adobe/serializable.hpp

```diff
@@ -63,10 +66,10 @@ template <class T>
 inline typename std::enable_if<!has_ostream_insertion<T>::value>::type
 ostream_insertion(std::ostream&, const T&) {}
 
-#else // __cplusplus < 202002L
+#else  // __cplusplus < 202002L
 
 template <class T>
-void ostream_insertion(std::ostream& s, const T& x) {
+inline void ostream_insertion(std::ostream& s, const T& x) {
```

Comment:
Why this change?

Reply:
"inline"? Because it is an empty or single-line function in a header. It doesn't mean much for a template.